moped-editor/src/views/projects/projectView/SignalComponentAutocomplete.js

```diff
@@ -12,7 +12,7 @@ import {
   renderSignalInput,
 } from "src/utils/signalComponentHelpers";
 const SOCRATA_ENDPOINT =
-  "https://data.austintexas.gov/resource/p53x-x73x.geojson?$select=signal_id,location_name,location,signal_type&$order=signal_id asc&$limit=9999";
+  "https://data.austintexas.gov/resource/p53x-x73x.geojson?$select=signal_id,location_name,location,signal_type,id&$order=signal_id asc&$limit=9999";
```

Comment:
since you have this code open...i think we should move SOCRATA_ENDPOINT to [signalComponentHelpers](https://github.com/cityofaustin/atd-moped/blob/main/moped-editor/src/utils/signalComponentHelpers.js). 100% my fault for being lazy and not doing that in the first place.